sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/options/BlobBeginCopyOptions.java

```diff
@@ -26,6 +26,7 @@
     private RequestConditions sourceRequestConditions;
     private BlobRequestConditions destinationRequestConditions;
     private Duration pollInterval;
+    private Boolean destinationSealed;
```

Comment:
I think "sealDestination" makes more sense to me because it's something that will happen in the future whereas "destinationSealed" sounds like it's commenting on the past/current state. I'm not really sure what you would use for the getter, though, because "isSealDestination" doesn't really work. Also not sure if this was already resolved in .NET

Reply:
modified